---
title: 'Summer Undergraduate Researcher'
timeperiod: 'June 2012 â€“ August 2012 (3 months)'
company: 'Boyce Thompson Institute, Ithaca, NY'
publishDate: 'Jun 25 2012'
seo:
  image:
    src: '/project_5.png'
---

<!---
```mermaid
%%{
  init: { 
    "theme": "base",
    'themeVariables': {
      'primaryColor': '#fff',
      'primaryTextColor': '#000000',
      'primaryBorderColor': '#b7b7b7',
      'lineColor': '#b7b7b7'
    }
 }
}%%
graph TD;
    A[Clean the raw data by removing barcode and adapters]-> B[Align reads to the virus database];
    B->C[Contigs correction];
    C->D[Remove redundant sequences];
    A -> E[Assemble contigs using velvet assembly];
    E -> F[Blast against non-redundant protein Sequences]
    F -> G[Compare results back to virus genome contigs]

    subgraph ALIGNMENT
            B
            C
            D
          end

    subgraph ASSEMBLY
            E
            F
            G
          end
```
-->            
<p align="center">
<img class="block dark:hidden" src="/project-5.png" alt="modifersimage" width="350"/>
<img class="hidden dark:block" src="/project-5_invert.png" alt="modifersimage" width="350"/>
</p>


**Figure**. The pipeline I used for alignment of known sweet potato viruses genomes (left) and assembly of novel ones (right).

## Summary:

*  [Improved methods to classify known and novel virus siRNA sequences from sweet potato samples more effectively](http://bioinfo.bti.cornell.edu/virome)

## Relevant Publications

* [Gutierrez, D., Tachin, M., Schulz, S., Miano, D., Ndunguru, J., Mukassa, S., ... & Kreuze, J. F. (2012). Determining the pan-African sweetpotato virome: Understanding virus diversity, distribution and evolution and their impacts on sweetpotato production in Africa.](https://cgspace.cgiar.org/server/api/core/bitstreams/4947f05c-9ebe-4d8e-9ca8-d17b3e13708d/content)